src/Core/IActiveSolutionTracker.cs

```diff
@@ -19,7 +19,6 @@
  */
 
 using System;
-using Microsoft.VisualStudio.Shell.Interop;
 
 namespace SonarLint.VisualStudio.Integration
```

Comment:
Moved to Core so I could use it in Typescript proj, I'll open a separate PR for namespace change